### PR TITLE
UIOR-468: Add missing translations

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**
## Purpose
Add missing translations for rendering dropdown list of record fields (for UIDATIMP-330)
https://issues.folio.org/browse/UIOR-468

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by aliaksei-chumakou** [comment]
I'm sorry, but I don't see any of these records are used in the code. What's the matter of adding them to the localization files?

**Comment by Vladyslav-Velytskyi** [comment]
Hi @aliaksei-chumakou!  These translations will be used in scope of story UIDATIMP-330, they are needed to display FOLIO records' fields list. List of required fields is attached to the story

File: translations/ui-orders/en.json [hunk 1]
```diff
@@ -773,5 +773,28 @@
...
+  
+  "createdDateAndTime": "Created date and time",
+  "createdSource": "Created source",
+  "fund.fundIdentifier": "Fund ID",
+  "fund.selectAdjustment": "Select adjustment",
+  "notes.date": "Date",
+  "notes.type": "Type",
+  "poAdministrativeData": "PO admin data",
+  "poPurchaseOrder": "PO purchase order",
+  "polAdministrativeData": "POL admin data",
```
Comment (mariia-aloshyna) [inline translations/ui-orders/en.json]: Rename "pol-" to "poLine-" and put it to the proper area, please
```suggestion
  "poLine.AdministrativeData": "POL admin data",
```

File: translations/ui-orders/en.json [hunk 2]
```diff
@@ -773,5 +773,28 @@
...
+  "polItemDetails": "POL item details",
+  "polDetails": "POL details",
+  "polCostDetails": "POL cost details",
+  "polFundDistribution": "POL fund distribution",
+  "polLocation": "POL location",
+  "polPhysicalResourceDetails": "POL physical resource details",
+  "polEResourceDetails": "POL e-resource details",
+  "polVendor": "POL vendor",
+  "polNotes": "POL notes",
```
Comment (mariia-aloshyna) [inline translations/ui-orders/en.json]: Rename it and put to the proper area, please

File: translations/ui-orders/en.json [hunk 3]
```diff
@@ -773,5 +784,17 @@
...
-  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}"
+  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}",
+  
+  "createdDateAndTime": "Created date and time",
+  "createdSource": "Created source",
+  "fund.fundIdentifier": "Fund ID",
+  "fund.selectAdjustment": "Select adjustment",
+  "notes.date": "Date",
+  "notes.type": "Type",
+  "poAdministrativeData": "PO admin data",
```
Comment (mariia-aloshyna) [inline translations/ui-orders/en.json]:
```suggestion
  "order.adminData": "PO admin data",
```

File: translations/ui-orders/en.json [hunk 4]
```diff
@@ -773,5 +784,17 @@
...
+  "poPurchaseOrder": "PO purchase order",
```
Comment (mariia-aloshyna) [inline translations/ui-orders/en.json]:
```suggestion
  "order.purchaseOrder": "PO purchase order",
```

File: translations/ui-orders/en.json [hunk 5]
```diff
@@ -773,5 +784,17 @@
...
 
   "poLine": "PO line",
-  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}"
+  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}",
+  
+  "createdDateAndTime": "Created date and time",
+  "createdSource": "Created source",
+  "fund.fundIdentifier": "Fund ID",
+  "fund.selectAdjustment": "Select adjustment",
+  "notes.date": "Date",
```
Comment (mariia-aloshyna) [inline translations/ui-orders/en.json]:
```suggestion
  "order.notes.date": "Date",
```

File: translations/ui-orders/en.json [hunk 6]
```diff
@@ -773,5 +784,17 @@
...
+  "notes.type": "Type",
```
Comment (mariia-aloshyna) [inline translations/ui-orders/en.json]:
```suggestion
  "order.notes.type": "Type",
```

**Comment by aliaksei-chumakou** [comment]
> Hi @aliaksei-chumakou! These translations will be used in scope of story UIDATIMP-330, they are needed to display FOLIO records' fields list. List of required fields is attached to the story

I see no connection here. How translations in ui-orders connected to code in your separate module/component? AFAIK, translations should live within the code, inside the same module. Please correct me if I'm wrong

**Pull request merged**